content/60.methods.md

```diff
@@ -123,24 +130,50 @@ Because in this implementation of Manubot, most collaborators were writing and e
+Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.
+<!-- To Do: Number of paper issue and number of unique contributors?-->
+
+Currently, three manuscripts associated with this project have been submitted to _mSystems_, with one accepted for publication [@arxiv:2102.02250] and two under revision [@arxiv:2103.02723; @arxiv:2102.01521].
+An additional four manuscripts are in preparation.
+All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they will continue to be updated as more information becomes available.
+These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@arxiv:2102.01521] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@arxiv:2103.02723], nutraceutical therapeutics [@arxiv:2102.02250], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing variable effects on patients [@individual-inequality].<!--To Do: add these additional individual citations-->
+To date, [number] authors are associated with the consortium.<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
+Efforts to integrate with existing projects providing support for undergraduate students during COVID-19 were also successful.
+Appendix A contains summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine [@url:https://github.com/ismms-himc/covid-19_sinai_reviews; @doi:10.1038/s41577-020-0319-0].
+Additionally, two of the consortium authors were undergraduate students recruited through the American Physician Scientist Association's Virtual Summer Research Program [@url:https://www.physicianscientists.org/page/summer-research-pilot-program].
+Thus, the consortium is expected to be successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.
+
+#### Integration of COVID-19 Resources with Manubot
+
+Data were integrated into the manuscripts from a number of sources.
+The workflow file is available from <https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml> and the scripts are available from <https://github.com/greenelab/covid19-review/tree/external-resources>.
```

Comment:
I would stick our code availability in METHODS.

Reply:
Excellent, I'll move it up in my next commit!